schema.js

```diff
@@ -1,34 +1,28 @@
 const axios = require('axios');
-const { GraphQLObjectType, GraphQLFloat, GraphQLList, GraphQLSchema, GraphQLString, GraphQLScalarType, GraphQLInt} = require('graphql');
+const { GraphQLObjectType, GraphQLFloat, GraphQLSchema, GraphQLString} = require('graphql');
 
 // Bitcoin data type
-const rateData = new GraphQLObjectType ({
-  name: 'Rates',
+const CurrencyRateType = new GraphQLObjectType({
```

Comment:
I changed the type here. Here I included what I wanted to output: the currency code, currency symbol, and the rate float. I am going to later down the file filter the API output to match this type.